sailor/sap_iot/wrappers.py

```diff
@@ -18,12 +18,12 @@
 from plotnine.scales import scale_x_datetime
 from sklearn.preprocessing import StandardScaler
 
+import sailor.assetcentral.indicators as ac_indicators
 from ..utils.plot_helper import _default_plot_theme
 from ..utils.timestamps import _any_to_timestamp, _calculate_nice_sub_intervals
-from ..assetcentral.indicators import AggregatedIndicator, AggregatedIndicatorSet
 
 if TYPE_CHECKING:
-    from ..assetcentral.indicators import IndicatorSet
+    from ..assetcentral.indicators import IndicatorSet, AggregatedIndicatorSet
```

Comment:
I don't think the AggregatedIndicatorSet is used as a type-hint anywhere?
It is however used in an `isinstance`-check in line 247 (can't comment there). that line is now missing an `ac_indicators.`